src/types/alerts.ts

```diff
@@ -0,0 +1,19 @@
+export namespace Alerts {
+  export interface AlertConfig {
```

Comment:
Nit/Minor - do we need to prepend `Alert` here? Since it's already in a namespace, you can reference it like so: `Alerts.Config`.